### DISCUSSION TITLE
GrugBus, powered by pymodbus

### DISCUSSION THREAD

**Discussion by peufeu2** [pymodbus-dev/pymodbus#1293] [General]
Greetings. Just shilling for my latest project which is an abstraction layer on top of pymodbus allowing fancy typed registers, bulk modbus transactions, and other stuff you may find interesting.

https://github.com/peufeu2/GrugBus

**Comment by janiversen** [comment]
Good luck with your project, I hope this means we can expect some pull requests from you solving bugs and/or adding functionality to pymodbus.
